plugins/commands/analyzer/src/main/kotlin/AnalyzerCommand.kt

```diff
@@ -228,5 +228,9 @@ class AnalyzerCommand : OrtCommand(
         val issues = analyzerRun.result.getAllIssues().flatMap { it.value }
         SeverityStatsPrinter(terminal, resolutionProvider).stats(issues)
             .print().conclude(ortConfig.severeIssueThreshold, 2)
+
+        for (packageManager in info.managedFiles.keys) {
+            packageManager.close()
```

Comment:
What I don't like so much about this approach is that every downstream user of `Analyzer.analyze()` has to call `close()` itself. I wonder whether a feasible / better approach would be for the respective package manager to override the existing `afterResolution()` and close their resources there.

Reply:
I understand your point, the downside is that the Analyzer is fed with a a list of managedFiles with their associated PackageManager. Thus when the Analyzer would close the PackageManagers after it has finished analysis, the caller would not be able to use the package managers anymore in a safe way as they are already closed. This transfer of ownership would have to be documented imho, but I am indifferent where the closing should happen, unless it happens ofc :-)